src/main/java/org/toradocu/extractor/ParamTag.java

```diff
@@ -65,20 +66,15 @@ public int hashCode() {
   /**
    * Returns a string representation of this param tag. The returned string is in the format "@param
    * PARAMNAME COMMENT" where PARAMNAME is the fully qualified name of the parameter in the param
-   * tag and COMMENT is the text of the comment in the param tag. If translation has been attempted
-   * on this tag, then the returned string is also appended with " ==&gt; CONDITION" where CONDITION
-   * is the translated condition as a Java expression or the empty string if translation failed.
+   * tag and COMMENT is the text of the comment in the param tag. If a translation of this tag is
```

Comment:
"the text of the comment in the param tag": with or without inline tags and code markers?